packages/webpack-stats-differ/src/diffAssets.ts

```diff
@@ -131,26 +132,18 @@ const diffWebpackAssets = ({ name, a, b }: Paired): AssetStats => {
   const isSizeIncrease = sizeDiff > 0;
   const isAssetRemoved = sizeB === 0;
   const isAssetAdded = sizeA === 0;
-  return isSignificantDifference(sizeDiff)
-    ? {
+  const isKeyAsset = b?.chunkNames?.join("").includes("⭐") || false;
```

Comment:
I doubt anyone is using this outside of our internal, but this is going to be really confusing if anyone is.

I am not sure how up-to-date the webpack stats plugin in the ardiffact repo (https://github.com/microsoft/ardiffact/blob/main/packages/bundle-stats-webpack-plugin/README.md) is wrt to internal one but I imagine at some point we should update the ardiffact one with changes from that one, so that this makes sense and it's usable if anyone does want to use externally.

Reply:
Ditto